management_app/schema.sql

```diff
@@ -103,4 +103,206 @@ VALUES
   ('Category 1', 1.25),
   ('Category 2', 1),
   ('Category 3', 0.75),
-  ('Category 4', 0.25)
\ No newline at end of file
+  ('Category 4', 0.25);
+
+-- Course Catalog
```

Comment:
I've already add the course table initialization in the catalog function in the course.py file (in the latest commit). We should pick one of them. Which one do you prefer?

Reply:
I think initializing in the db script is better that way we don't have to check if the database already has the course catalog data every time the catalog endpoint is requested.